src/sdl2.nim

```diff
@@ -1120,7 +1120,14 @@ proc setClipRect*(surface: SurfacePtr; rect: ptr Rect): Bool32 {.
   importc: "SDL_SetClipRect".}
 proc getClipRect*(surface: SurfacePtr; rect: ptr Rect) {.
   importc: "SDL_GetClipRect".}
-
+  
+proc rendererSetClipRect*(renderer: RendererPtr; rect: ptr Rect): cint {.
```

Comment:
Nice, but this should be called ``setClipRect``. And likewise for your other additions.